### PR TITLE
Bugfix/orphan pt_BR properties

### PR DESCRIPTION
**Pull request by glasswalk3r**
There is no Jenkins issue associated.

Removed deprecated tokens using `./translation-tool.pl --lang=pt_BR --remove`. Even though executed twice, the result still shows orphans:

```
TOTAL: Files: 663 Keys: 2062 Done: 1590(77.11%)
       Missing: 442(21.44%) Orphan: 9(0.44%) Empty: 2(0.10%) Same: 19(0.92%) NoJenkins: 0(0.00%)
```

### Proposed changelog entries

N/A

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [ ] (If applicable) Jira issue is well described
- [X] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change) and are in the imperative mood. [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadoc, as appropriate. 
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are accurate, human-readable, and in the imperative mood
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

**Review by basil — APPROVED** [review]

File: core/src/main/resources/hudson/slaves/JNLPLauncher/help_pt_BR.properties
Comment (timja) [inline core/src/main/resources/hudson/slaves/JNLPLauncher/help_pt_BR.properties:24]: is this right? it seems to be just cutting off the text. there's a \ without any continuation

File: core/src/main/resources/jenkins/diagnostics/ControllerExecutorsAgents/message_pt_BR.properties
Comment (timja) [inline core/src/main/resources/jenkins/diagnostics/ControllerExecutorsAgents/message_pt_BR.properties:25]: is this right? it seems to be just cutting off the text. there's a \ without any continuation

File: core/src/main/resources/hudson/slaves/JNLPLauncher/help_pt_BR.properties
Comment (glasswalk3r) [reply core/src/main/resources/hudson/slaves/JNLPLauncher/help_pt_BR.properties:24]: You're right... I checked against the original file and looks like translation_tool.pl messed up in this one. Compare with the original file in English:

```
blurb=\
  Allows an agent to be connected to the Jenkins controller whenever it is ready.<br>\
  In one mode, <a href="https://en.wikipedia.org/wiki/Java_Web_Start" rel="noopener noreferrer" target="_blank">Java Web Start</a> is used. \
  In this case, a JNLP file must be opened on the agent machine, \
  which will establish a TCP connection to the Jenkins controller. \
  (Other launch methods use a JNLP file but not Java Web Start, or do not use a JNLP file at all.)<br>\
  This means that the agent need not be reachable from the controller; \
  the agent just needs to be able to reach the controller. \
  If you have enabled security via the <i>Configure Global Security</i> page, \
  you can customize the port on which the Jenkins controller will listen for incoming agent connections.<br>\
  By default, the agent will launch a GUI, but it is also possible to run \
  an agent without a GUI, for example as a Windows service.
```
Maybe because de sentence starts with a backslash (`\`)?
I'll fix that manually and send again.

File: core/src/main/resources/jenkins/diagnostics/ControllerExecutorsAgents/message_pt_BR.properties
Comment (glasswalk3r) [reply core/src/main/resources/jenkins/diagnostics/ControllerExecutorsAgents/message_pt_BR.properties:25]: Same thing here... the last sentence was removed although the original has it:

```
ExecutorsWarning=Building on the built-in node can be a security issue. \
  You should set the number of executors on the built-in node to 0. \
  See <a href="https://www.jenkins.io/redirect/building-on-controller/" target="_blank" rel="noopener noreferrer">the documentation</a>.
# TODO(terminology) rename redirect URL
```

Comment (glasswalk3r) [reply core/src/main/resources/jenkins/diagnostics/ControllerExecutorsAgents/message_pt_BR.properties:25]: And after fixed, I executed the translation_tool.pl again:

```
$ git diff
diff --git a/core/src/main/resources/jenkins/diagnostics/ControllerExecutorsAgents/message_pt_BR.properties b/core/src/main/resources/jenkins/diagnostics/ControllerExecutorsAgents/message_pt_BR.properties
index 8df5b1eea3..7d609c3194 100644
--- a/core/src/main/resources/jenkins/diagnostics/ControllerExecutorsAgents/message_pt_BR.properties
+++ b/core/src/main/resources/jenkins/diagnostics/ControllerExecutorsAgents/message_pt_BR.properties
@@ -22,4 +22,3 @@
 
 ExecutorsWarning=Construir no n\u00F3 embutido pode ser um problema de seguran\u00E7a. \
 Voc\u00EA deveria configurar o n\u00FAmero de executores no n\u00F3 embutido para 0. \
-Vide <a href="https://www.jenkins.io/redirect/building-on-controller/" target="_blank" rel="noopener noreferrer">a documenta\u00E7\u00E3o</a>.
```

It just removed the last line again.

File: core/src/main/resources/jenkins/diagnostics/ControllerExecutorsNoAgents/message_pt_BR.properties
Comment (timja) [inline core/src/main/resources/jenkins/diagnostics/ControllerExecutorsNoAgents/message_pt_BR.properties:26]: same problem here and in the next few files

Comment (glasswalk3r) [reply core/src/main/resources/jenkins/diagnostics/ControllerExecutorsNoAgents/message_pt_BR.properties:26]: Indeed. I reviewed and fix all files again.
This looks like a bug to me with `translation_tool.pl`, it is not only removing sentences it shouldn't, it is also messing up with the two spaces prefix when the previous line ends with a `\`.
If you agree, I can open a Jira ticket to address that.

**Review by timja — APPROVED** [review]

**Comment by timja** [comment]
This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Pull request merged**
